app/components/c-product-detail.vue

```diff
@@ -61,13 +62,22 @@
          *
          * @returns  {Object}  product - Single product from the store
          */
-        product: 'product/getProduct'
+        product: 'product/getProduct',
```

Comment:
- [ ] I think we should not use `get...` for the getters, because they will all start with this. Better just use the name of the data they return. `product` in this case.